VK/VK/Animation/StartViewController.swift

```diff
@@ -0,0 +1,24 @@
+// StartViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// StartVC with animation
```

Comment:
```suggestion
/// StartVC with animation
```
Не понятно по названию что куда зачем